mypyc/test-data/irbuild-frozenset.test

```diff
@@ -0,0 +1,536 @@
+[case testNewFrozenSet]
```

Comment:
The irbuild tests are maybe too verbose -- they may need to be updated frequently as the details of emitter IR change. My suggestion would be to avoid some duplication and to move some thing to run tests if they don't otherwise have test coverage.

Reply:
Tbh for the first implementation I mostly copied the `set` test cases and adjusted them slightly for `frozenset`. Your suggestion does make sense though. Moved some of the `fromIterable` tests to `run_sets.test`.

Reply:
Yeah some of the existing IR build tests are overly verbose.